### PR TITLE
Update Tutorial 4 code dependencies and deploy contracts

### LINKED ISSUE #742: Update tutorial 4 code and redeploy contracts
- [ ] Deploy tutorial 4 contract 
- [ ] Update dependancies
- [ ] Update all files and configs to reflect current scaffolds generated from zkApp-cli and o1js
- [ ] Confirm tutorial 4 is working
- [ ] Deploy tutorial 4 UI to GithubPages

### PR DESCRIPTION
**Pull request by ymekuria**
Closes #742

### CODE REVIEW

File: examples/zkapps/04-zkapp-browser-ui/contracts/config.json [hunk 1]
```diff
@@ -1,11 +1,14 @@
 {
   "version": 1,
-  "networks": {
+  "deployAliases": {
```
Comment (barriebyron) [inline examples/zkapps/04-zkapp-browser-ui/contracts/config.json:3]: love this change!

File: examples/zkapps/04-zkapp-browser-ui/contracts/config.json [hunk 2]
```diff
@@ -1,11 +1,14 @@
...
     "berkeley": {
       "url": "https://proxy.berkeley.minaexplorer.com/graphql",
       "keyPath": "keys/berkeley.json",
-      "fee": "0.1",
+      "feepayerKeyPath": "This is specific to each developer and can be generated with the zkApp-cli. Update with your feepayer keypath to run this tutorial example",
```
Comment (barriebyron) [inline examples/zkapps/04-zkapp-browser-ui/contracts/config.json:7]:
```suggestion
      "feepayerKeyPath": "This is specific to each developer and can be generated with the zkApp CLI. Update with your feepayer keypath to run this tutorial example",
```

**Review by barriebyron — APPROVED** [review]
I did not evaluate the code changes, but I like the updates

**Comment by barriebyron** [comment]
@ymekuria  thanks for these updates. Did you want me to test these changes? (if so, can you guide me so that I can be sure to use this branch?)

**Comment by ymekuria** [comment]
> @ymekuria thanks for these updates. Did you want me to test these changes? (if so, can you guide me so that I can be sure to use this branch?)

These changes have been tested but the current o1js version is different than the one used in this tutorial. I will update the version and test again.

**Pull request merged**